app/persistence/transaction/transaction.go

```diff
@@ -51,12 +56,12 @@ func (tr *TransactionRepository) GetByTransactionId(transactionId string) (*Tran
 	return tx, nil
 }
 
-func (tr *TransactionRepository) GetPendingOrSubmittedTransactions() ([]*Transaction, error) {
+func (tr *TransactionRepository) GetIncompleteTransactions() ([]*Transaction, error) {
```

Comment:
nit: I think that the name of the function now is not 100% correct. Maybe it would be better to rename it to `GetPendingOrSignatureSubmittedTX`